### PR TITLE
[bitnami/etcd] add loadBalancerSourceRanges

### PR DESCRIPTION
**Pull request by rico-bincentive**
**Description of the change**

add `loadBalancerSourceRanges` new option.

**Benefits**

Restrict only accessible traffic can access etcd service from the load balancer.

**Possible drawbacks**

no.

**Applicable issues**

  - feature request #3629

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/etcd/templates/svc.yaml [hunk 1]
```diff
@@ -10,6 +10,12 @@ spec:
   type: {{ .Values.service.type }}
   {{- if and (eq .Values.service.type "LoadBalancer") (not (empty .Values.service.loadBalancerIP)) }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
+  {{- end }}
+  {{- if and (eq .Values.service.type "LoadBalancer") (not (empty .Values.service.loadBalancerSourceRanges)) }}
+  loadBalancerSourceRanges:
+  {{- with .Values.service.loadBalancerSourceRanges }}
```
Comment (javsalgar) [inline bitnami/etcd/templates/svc.yaml:16]: Is the `with` instruction necessary? wouldn't using toYaml with the array be enough?

File: bitnami/etcd/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: etcd
-version: 4.10.1
+version: 4.10.2
```
Comment (javsalgar) [inline bitnami/etcd/Chart.yaml:3]: As it is adding a feature, please bump the minor version

File: bitnami/etcd/templates/svc.yaml (hunk 1 above)
Comment (rico-bincentive) [reply bitnami/etcd/templates/svc.yaml:16]: Thank for the feedback. You are right. But I saw some of chart use this pattern, some of not.
use `with`:
https://github.com/bitnami/charts/blob/master/bitnami/redis/templates/redis-master-svc.yaml#L23-L28
https://github.com/bitnami/charts/blob/master/bitnami/influxdb/templates/influxdb/service.yaml#L16-L21
just use `toYaml`:
https://github.com/bitnami/charts/blob/master/bitnami/external-dns/templates/service.yaml#L19-L21
https://github.com/bitnami/charts/blob/master/bitnami/thanos/templates/compactor/service.yaml#L19-L21

Comment (javsalgar) [reply bitnami/etcd/templates/svc.yaml:16]: Yes, we are trying to normalize it Could you go for toYaml instead?

Comment (rico-bincentive) [reply bitnami/etcd/templates/svc.yaml]: sure

**Review by javsalgar — APPROVED** [review]
LGTM! Thanks

**Pull request merged**